cmd/validator/validator.go

```diff
@@ -13,27 +13,31 @@ The flags are:
 		The search path for configuration files
     -exclude-dirs string
     	Subdirectories to exclude when searching for configuration files
+	-reporter string
+		Format of printed report. Currently supports standard, JSON.
 */
 
 package main
 
 import (
 	"flag"
 	"fmt"
-	"github.com/Boeing/config-file-validator/pkg/cli"
 	"log"
 	"os"
 	"strings"
+
+	"github.com/Boeing/config-file-validator/pkg/cli"
 )
 
 // Parses, validates, and returns the flags
 // flag.String returns a pointer
 // If a required parameter is missing the help
 // output will be displayed and the function
 // will return with exit = 1
-func getFlags() (*string, *string, int) {
+func getFlags() (*string, *string, *string, int) {
 	searchPathPtr := flag.String("search-path", "", "The search path for configuration files")
 	excludeDirsPtr := flag.String("exclude-dirs", "", "Subdirectories to exclude when searching for configuration files")
+	reportTypePtr := flag.String("reporter", "standard", "Format of the printed report")
```

Comment:
Identify all available options in flag description instead of just "Format of the printed report".  For example, "Format of the printed report. Options are `standard` and `json`"